src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    KNOWN_PERMISSIONS = WEBEXT_PERMISSIONS_DICT
+    name = models.CharField(max_length=255, unique=True, null=False)
+    _enum = models.IntegerField(default=WEBEXT_UNKNOWN.id, db_column='enum')
```

Comment:
Assuming we do keep the `ManyToManyField` approach (see above) I'm not a fan of keeping yet another set of ids around. We could just use the name to find the description + set some `choices` on the `name` field, which would give us `get_name_display()` for free.